.github/workflows/ci.yml

```
@@ -23,27 +23,23 @@ jobs:
       id: mafia
       run: |
         set -o pipefail
-        export MAFIA_VERSION=$(head -n1 RELEASE/scripts/autoscend.ash | grep -o '[0-9]\+')
-        if [[ -z "$MAFIA_VERSION" ]]; then
-          echo "Couldn't determine required mafia version!"
-          exit 1
-        fi
-        export MAFIA_BUILD=$(curl --fail --silent --globoff 'https://ci.kolmafia.us/job/Kolmafia/api/json?tree=builds[number,artifacts[relativePath]]' | jq '[.builds[]] | map(select(.artifacts[].relativePath | contains(env.MAFIA_VERSION))) | .[] | .number' | head -n1)
+        export MAFIA_BUILD=$(curl --fail --silent --globoff 'https://ci.kolmafia.us/job/Kolmafia/api/json?tree=lastCompletedBuild[changeSet[items[revision]]]' | grep -o '[0-9]\+')
```

Try this so you can keep using jq instead of grep:
```suggestion
        export MAFIA_BUILD=$(curl --fail --silent --globoff 'https://ci.kolmafia.us/job/Kolmafia/api/json?tree=lastCompletedBuild[changeSet[items[revision]]]' | jq '.lastCompletedBuild.changeSet.items[0].revision')
```